lib/matplotlib/legend.py

```diff
@@ -976,9 +976,11 @@ def get_title(self):
         'Return the `.Text` instance for the legend title.'
         return self._legend_title_box._text
 
-    def get_window_extent(self, *args, **kwargs):
+    def get_window_extent(self, renderer=None):
```

Comment:
That's technically a backward incompatible change (although, as I've mentioned in a previous PR doing something similar, I think this is  a change that is worth making.)